apps/backend/app/api/routes/upload.py

```
@@ -188,7 +181,5 @@ async def upload_file(
         }
 
     except Exception as e:
-        # Log the full traceback
-        logger.error(f"Error processing file: {str(e)}")
-        traceback.print_exc()
+        logger.exception(f"Error processing file: {str(e)}")
```

Ruff (0.15.15)

[warning] Use explicit conversion flag

Replace with conversion flag

(RUF010)